script/QA/QA_18.txt

```diff
@@ -31,17 +33,18 @@ I am your commentator, Eco-Arc.
 [sha:2ad378be0102fad4e7863418e67122b6ee9eefa7]{
 -- Page 115, Offset 43527.
 -- <隅|コーナー>に咲く一輪の花、
-And I am the flower blooming from a corner,
+And I am the flower that blooms%{n}
+in the corner:
 }
 [sha:ff51fc07ae23eecb72f925ce1855969836a60dc2]{
 -- Page 115, Offset 43528.
 -- ネコアルクでお送りします。
-otherwise known as Neco-Arc.
+ Neco-Arc.
 }
 [sha:b22dcc7526d79e4bf0a1e10733926494125f770c]{
 -- Page 116, Offset 43529.
 -- って、シエルはドコいったのかニャ？
-By the way, where did Ciel go nya?
+So uh, where'd Ciel disappear to meow?%{n}
 }
 [sha:35f6ddc645d466aa523d75a3e067ef0be548513d]{
 -- Page 116, Offset 43530.
```

Comment:
Using "you" like this is a Neco-Arc special. Despite her obnoxious omnipresence in Melty I've yet to decide on a consistent way of rendering it. It'd probably be good to get this "character trait" consistent.

Reply:
I'd suggest another font or something but QA uses a weird fucked up OTF font that I wasn't able to successfully add new glyphs to last I tried

Reply:
Maybe a single upper-case "U". Might be dumb enough, and is adaptable to all game engines (MBTL cannot use any special font effects)

Reply:
Works for me, though kinda awkward for e..g.
> んー、なんでそこまで知ってるのかコワイけど、その高い知性、youやっぱりアタシじゃ～ん☆
> I'm a little scared to ask how u know all that, but you must really be me if you're that smart ☆

where we have to use a bunch more pronouns in EN. I guess just triple-u?

Reply:
"how U know all that", it would be, AHEM.  But no, just use the one, I think. it's like an emphasis word. The listener will only hear it once.